app/code/Magento/InventorySourceSelection/Model/DistanceProvider/Offline/GetDistance.php

```diff
@@ -0,0 +1,38 @@
+<?php
+/**
+ * Copyright © Magento, Inc. All rights reserved.
+ * See COPYING.txt for license details.
+ */
+declare(strict_types=1);
+
+namespace Magento\InventorySourceSelection\Model\DistanceProvider\Offline;
+
+use Magento\InventorySourceSelection\Model\DistanceProvider\GetDistanceInterface;
+use Magento\InventorySourceSelectionApi\Model\Request\LatLngRequestInterface;
+
+/**
+ * @inheritdoc
+ */
+class GetDistance implements GetDistanceInterface
+{
+    private const EARTH_RADIUS = 6371000.0;
```

Comment:
it's better to specify at least in comment that this value is in kilometer, as guys who are not familiar with metric system could be confused